.atomist/editors/JavaTypeUsageTestTypeScriptTest.ts

```diff
@@ -0,0 +1,61 @@
+import {Project} from '@atomist/rug/model/Core'
+import { PathExpressionEngine , Microgrammar } from '@atomist/rug/tree/PathExpression'
+import {Editor} from '@atomist/rug/operations/Decorators'
+
+/*
+ * This is a single-use editor
```

Comment:
I kinda like it. It shows the power of our technology. We may delete this editor in time, but it's a nice moment in history.